cdc/sink/metrics/metrics.go

```diff
@@ -17,6 +17,9 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+// rowSizeLowBound is set to 1MB, only track data event with size not smaller than it.
+const rowSizeLowBound = 1024 * 1024
```

Comment:
```suggestion
const rowSizeLowBound = 128 * 1024
```

Could you set to 128KB?